app/jobs/monitor_job.rb

```diff
@@ -1,37 +1,26 @@
 require 'monitoring/monitor'
 require 'monitoring/librato_notifier'
+require 'monitoring/aws_notifier'
 require 'monitoring/failed_job_by_class_check'
 require 'monitoring/failed_job_check'
 require 'monitoring/queue_size_check'
 require 'monitoring/stale_worker_check'
+require "active_support/core_ext/string/inflections.rb"
 
 class MonitorJob
 
-  def self.perform(checker_name, error_handling = :raise)
-    checker,notifier = CHECKS_AND_NOTIFIERS.fetch(checker_name.to_sym).()
+  def self.perform(checker_klass_name,
```

Comment:
I had to make this class much more generic so that it could be parameterized with classes and args.